packages/react-components/src/utils/sort.ts

```diff
@@ -1,13 +1,12 @@
-import { DataPoint, Primitive } from '../common/dataTypes';
+import { DataPoint, Primitive } from '@iot-app-kit/core';
 
 /**
  * Sorts points in order of their points values.
  * Places objects with no point at the end of the list.
  */
-// TODO: rename to 'sortPoints'
-export const sortTooltipPoints =
-  (attr: (point: DataPoint) => Primitive) =>
-  (a: { point?: DataPoint }, b: { point?: DataPoint }): number => {
+export const sortPoints =
+  <T extends Primitive>(attr: (point: DataPoint<T>) => T) =>
+  (a: { point?: DataPoint<T> }, b: { point?: DataPoint<T> }): number => {
```

Comment:
Nit: Can the type annotation be abstracted into a symbol? It's difficult to read.

Reply:
refactored to `PointWrapper<T>`